build/Dockerfile-setup-sqlserver

```diff
@@ -0,0 +1,11 @@
+FROM phusion/baseimage
```

Comment:
Please use debian:jessie image because all other containers are using it or better move mssql setup to setup container. Unfortunately official mssql container is available only as Ubuntu and RHEL based so this will be the only exception.